CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ x.x.x Release notes (yyyy-MM-dd)
 * Removed undocumented, outdated and unused `ListView` component exported via `realm/react-native`. See [@realm.io/react](https://www.npmjs.com/package/@realm.io/react) for a modern integration with React.
 * Sending the correct version when submitting analytics requests on Android, as a side-effect of [#4114](https://github.com/realm/realm-js/pull/4114/files#diff-58e627175b916d5dcb05c3c8bd5b10fb18cd72ce6e40e41d8d1c51e984fe32e8L73-R73).
 * Removed TypeScript dependency from individual sub-packages so all packages share the root version, and updated the root TypeScript version to `4.5.2`.
+* Consuming TypeScript directly from the integration test environments, instead of transpiling first.
 * Upgraded Realm Core from v11.6.1 to v11.7.0.
```

Comment:
It would be nice to have a line here mentioning the new metro-config

Reply:
```suggestion
* Adding a new private `@realm/metro-config` package to share this across any React Native app in our repo that reference other packages via symbolic links.
* Upgraded Realm Core from v11.6.1 to v11.7.0.
```